addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.NZD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.JPY" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.CNY" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.MYR" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.SGD" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.GBP" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.lang_en" model="res.lang">
```

Comment:
@mart-e @richard-willowit  I do not think there is an en_AU or en_NZ however notwithstanding this, this (may) create a nasty bug.

Firstly, what if both a US and Australian company use the same instance.  Second and more common, the date  fields use this locale in order to parse dates in the UI, however it takes no reference of these fields unless all characters are used.  

So for example 06/01/2010 would be saved as 6th January 2010, but 6/1/10 would be saved as first June.  I would really advise against changing the date_format of a language ever.   In any case en_GB works just fine except that now you have to deal with unnecessary translations on fields which themselves cause issues. At least this was the behaviour in v7, v8 I have not tested lately as I learnt the error of my ways.